news/2021-03-05-agm-results.md

```diff
@@ -0,0 +1,111 @@
+---
+title: AGM 2021 Results
+---
+
+It'll take more than a global pandemic to stop the HackSoc AGM!
+
+The new committee is:
+
+ - **Chair**: Ash Holland
+ - **Secretary**: Marks Polakovs & Daniel Scoffin
+ - **Treasurer**: Morgan McKay
+ - **Social Secretary**: C Lloyd
+ - **Press and Publicity Officer**: Chester Wringe
+ - **Academic Events Officer**: Aaron Christiansen
+ - **Infrastructure Officer**: Ezekiel Bethel
+ - **Ordinary Members**:
+   - Daniel Allinson
+   - Adam Birtles
+   - Jamie Quigley
+
+Candidates met on Google Meet to present their manifesti, while the meeting was streamed using our in-house RTMP service with a latency of around 10 seconds, and our brand-new Lightspeed service with less than a second of delay. Questions from members were taken via Slack. Frequent pauses were given to allow any livestream delay to catch up before moving on to the next item.
+
+## Amendments
+
+Two amendments were presented to the society, both of which were accepted. The passing of an amendment required the support of two-thirds of the members voting. You can learn more about the amendments in the linked GitHub issues.
+
+ - [Appendicies](https://github.com/HackSoc/constitution/pull/8)
+   - **Final vote:** 16 for (94%), 0 against, 1 abstaining
+   - **Passed**
+ - [Common Practice of the Committee](https://github.com/HackSoc/constitution/pull/11)
+   - **Final vote:** 17 for (100%), 0 against, 0 abstaining
+   - **Passed**
+
+## Committee Election Results
+
+All positions were voted on using the Single Transferable Vote (STV) method, which is equivalent to the Alternative Vote (AV) method for positions with one seat (all positions except Ordinary Members). All positions had the option to Re-Open Nominations (RON). Quorum was **12** based on our membership count at the time, which was achieved for all elections.
+
+Voters were given access to a Google Form for each election, which required them to be signed into their University account, and allowed paid members to cast at most one vote per election. The elections were open for ten minutes at a time, then closed to prevent further submissions. The results of the Google Forms were collected into a spreadsheet, which was then used to count the votes manually, or electronically in the case of the Ordinary Member election. Voters email addresses were compared against a list of HackSoc members, to ensure that all voters were eligible to vote. All votes will later be verified manually. 
```

Comment:
nit:

```suggestion
Voters were given access to a Google Form for each election, which required them to be signed into their University account, and allowed paid members to cast at most one vote per election. The elections were open for ten minutes at a time, then closed to prevent further submissions. The results of the Google Forms were collected into a spreadsheet, which was then used to count the votes manually, or electronically in the case of the Ordinary Member election. Voters' email addresses were compared against a list of HackSoc members, to ensure that all voters were eligible to vote. All votes will later be verified manually. 
```